_episodes/01-introduction.md

```diff
@@ -7,11 +7,10 @@ questions:
 - "How can we describe teaching and learning how to teach?"
 - "What common vocabulary do we need to talk about teaching and learning?"
 objectives:
-- "Describe the ways that learning to teach is comparable to a performing art."
+- "Describe how that learning to teach is like learning any other complex skill."
```

Comment:
Again -1, but I do propose a small edit:

"Describe the ways that learning to teach is comparable to *learning* a performing art."